src/app/query/common/market-data/market-data.hooks.ts

```diff
@@ -59,9 +58,9 @@ export function useCalculateBitcoinFiatValue() {
   const btcMarketData = useCryptoCurrencyMarketData('BTC');
 
   return useCallback(
-    (value: string | number) =>
+    (value: Money) =>
       baseCurrencyAmountInQuote(
-        createMoneyFromDecimal(isNumber(value) ? value : Number(value), 'BTC'),
+        createMoneyFromDecimal(value.amount, value.symbol, value.decimals),
```

Comment:
If alreaday `Money`, I don't need this, do I? Not sure I understand the naming of that function?

Reply:
O, I see, this takes the primary unit so `satToBtc` isn't needed? What does `Decimal` refer to in the naming bc both `createMoney` and `createMoneyFromDecimal` use decimals?

Reply:
`createMoneyFromDecimal(value.amount, value.symbol, value.decimals),` you can delete this entire line `baseCurrencyAmountInQuote` accepts money, and it already is that type

Reply:
decimal means you can go `createMoneyFromDecimal(0.1, 'BTC')`, whereas the other accepts sats